### ISSUE TITLE
If stream title contains a /, then the same won't be saved

### ISSUE THREAD

**Issue by HumboShot** [adriel/docker-streamlink-autodl#24]
I tired setting up this container, but it seems to fail on streams that contain a slash in the title, possibly because FFMPEG thinks it's a new directory

Stacktrace
```
UID : 1000 
GID : 1000
Saving config file.
Updated configuration.
Waiting for stream to go live.
Token missing, refreshing it.
2023/08/26 10:53:34 App Access Token: CENSORED
Token refreshed, now waiting for stream to go live.
mogra is LIVE!
user_name: MOGRA
user_id: 126482446
started_at: 2023-08-26T04:44:24Z
started_at_safe: 2023-08-26T04.44.24Z
game_name: Music
title: 8/26(sat) MOGRA 14th ANNIVERSARY PARTY DAY2
viewer_count: 1136
/home/download/MOGRA/Music/MOGRA 1693040074 2023-08-26T04.44.24Z 8/26(sat) MOGRA 14th ANNIVERSARY PARTY DAY2 [viewers 1136] (live_dl).mp4: No such file or directory
error: Error when writing to output: [Errno 32] Broken pipe, exiting
Stream ended, finished processing. Watching for next live event.
```

**Comment by adriel** [comment]
Oh sorry, I didn't have issue notifications turned on, so missed this.

But yeah, you're right, ffmpeg wasn't a fan of the `/`. It's been fixed in c641eece25341889e24778d725352e9c390e3670.

**Comment by HumboShot** [comment]
Thank you so much :)
